src/SFA.DAS.ApplyService.Data/ApplyRepository.cs

```diff
@@ -1130,6 +1130,80 @@ public async Task RemoveSectionCompleted(Guid applicationId, Guid applicationSec
             }
         }
 
+        public async Task<List<RoatpAssessorApplicationSummary>> GetNewAssessorApplications(string userId)
+        {
+            using (var connection = new SqlConnection(_config.SqlConnectionString))
+            {
+                return (await connection
+                    .QueryAsync<RoatpAssessorApplicationSummary>(
+                        @"SELECT 
+                            ApplicationId,
+                            org.Name AS OrganisationName,
+                            JSON_VALUE(apply.ApplyData, '$.ApplyDetails.UKPRN') AS Ukprn,
+                            JSON_VALUE(apply.ApplyData, '$.ApplyDetails.ReferenceNumber') AS ApplicationReferenceNumber,
+                            JSON_VALUE(apply.ApplyData, '$.ApplyDetails.ProviderRouteName') AS ProviderRoute,
+                            JSON_VALUE(apply.ApplyData, '$.ApplyDetails.ApplicationSubmittedOn') AS SubmittedDate,
+                            Assessor1Name,
+                            Assessor2Name
+	                      FROM Apply apply
+	                      INNER JOIN Organisations org ON org.Id = apply.OrganisationId
+	                      WHERE apply.DeletedAt IS NULL AND apply.GatewayReviewStatus = @gatewayReviewStatusApproved AND (Assessor1UserId IS NULL OR Assessor1UserId <> @userId AND Assessor2UserId IS NULL)
```

Comment:
I think when we refined the stories, we discussed the fact that someone could assign themselves to Assessor 2 before someone else took on the Assessor 1 role, does this need an extra check?